{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2017-1000158", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "CPython (aka Python) up to 2.7.13 is vulnerable to an integer overflow in the PyString_DecodeEscape function in stringobject.c, resulting in heap-based buffer overflow (and possible arbitrary code execution)", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 15:52:42 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2018-10-21", "cve_detail": "", "references": [{"URL": "https://www.debian.org/security/2018/dsa-4307", "Description": "DEBIAN:DSA-4307"}, {"URL": "https://security.gentoo.org/glsa/201805-02", "Description": "GENTOO:GLSA-201805-02"}, {"URL": "https://bugs.python.org/issue30657", "Description": "MISC:https://bugs.python.org/issue30657"}, {"URL": "https://lists.debian.org/debian-lts-announce/2017/11/msg00035.html", "Description": "MLIST:[debian-lts-announce] 20171124 [SECURITY] [DLA 1189-1] python2.7 security update"}, {"URL": "https://lists.debian.org/debian-lts-announce/2017/11/msg00036.html", "Description": "MLIST:[debian-lts-announce] 20171124 [SECURITY] [DLA 1190-1] python2.6 security update"}, {"URL": "https://lists.debian.org/debian-lts-announce/2018/09/msg00030.html", "Description": "MLIST:[debian-lts-announce] 20180925 [SECURITY] [DLA 1519-1] python2.7 security update"}, {"URL": "https://lists.debian.org/debian-lts-announce/2018/09/msg00031.html", "Description": "MLIST:[debian-lts-announce] 20180926 [SECURITY] [DLA 1520-1] python3.4 security update"}, {"URL": "http://www.securitytracker.com/id/1039890", "Description": "SECTRACK:1039890"}], "cve_repaire": "", "editor": "warcup", "add_time": "2017-11-17"}